src/Controller/IndexController.php

```diff
@@ -116,14 +116,18 @@ public function pastImportsAction()
     {
         if ($this->getRequest()->isPost()) {
             $data = $this->params()->fromPost();
-            $undoJobIds = [];
-            foreach ($data['jobs'] as $jobId) {
-                $undoJob = $this->undoJob($jobId);
-                $undoJobIds[] = $undoJob->getId();
+            if (isset($data['jobs'])) {
+                $undoJobIds = [];
+                foreach ($data['jobs'] as $jobId) {
+                    $undoJob = $this->undoJob($jobId);
+                    $undoJobIds[] = $undoJob->getId();
+                }
+                $message = new Message('Undo in progress in the following jobs: %s'  // @translate
+                    , implode(', ', $undoJobIds));
```

Comment:
As a matter of style, we wouldn't have a leading comma like this on a line, it should go back up to the previous line.

Reply:
Looking at the blame, that was something Patrick changed for whatever reason. I agree it looks weird, will fix

Reply:
Thanks. I didn't catch that it was just a movement of a pre-existing block... interesting that the style auto-fixer didn't catch it before.

However, your fix introduces a problem: you moved the `@translate` comment. Those are used by our system for picking up translatable strings, and the way they work is that the comment applies to the closest preceding string. So, moving the comment to the next line is marking the `', '` string as translatable, not the intended message.

Reply:
Ah OK good to know. I guess that's why he split the line up, but with the misplaced comma. hopefully fixed now